### PR TITLE
check if the offer has been issued on the home page and display appro…

### PR DESCRIPTION
**Pull request by soginga**
…priate message if it doesn't or url to it if does #305

### CODE REVIEW

File: webapp/src/pages/home/Home.js [hunk 1]
```diff
@@ -116,26 +124,31 @@ class Home extends Component {
...
                 </h4>
-                
-                {this.state.applicationStatus == "Submitted" && <p>
-                  Thank you! Your application has been received and is under review. You can expect to hear our decision mid-June.
-                </p>}
+
+                {this.state.applicationStatus == "Submitted" &&
+                <div>
+                  <p>
+                    Thank you! Your application has been received and is under review. You can expect to hear our decision mid-June.
```
Comment (avishkar58) [inline webapp/src/pages/home/Home.js]: Lets remove this line - so we just have information about the offer. Alternatively you could add a table with the event, application date and offer status

File: webapp/src/pages/home/Home.js [hunk 2]
```diff
@@ -116,26 +124,31 @@ class Home extends Component {
...
+                  </p>
+                  {this.state.offer !== null ? <p>There is an offer waiting for you, <NavLink to="/offer">click here</NavLink> to view it.</p>
+                  : <p>You are currently on the waiting list for the Deep Learning Indaba 2019. Please await further communication.</p>}
+                </div>}
                 {this.state.applicationStatus == "Withdrawn" && <p>
                   Your application has been withdrawn - you will not be considered for a place at the Indaba.
                 </p>}
                 {this.state.applicationStatus == "NOT Submitted" && <p>
                   You have NOT submitted your application! <NavLink to="/applicationForm">Complete and submit</NavLink> it before 26 May to be considered for a place at the Indaba.
```
Comment (avishkar58) [inline webapp/src/pages/home/Home.js]: Please remove the links to applicationForm as these are no longer relevant :)

Comment (soginga) [reply webapp/src/pages/home/Home.js]: what should be the remaining text there

> You have NOT submitted your application!
?

**Comment by avishkar58** [comment]
Yeah that one can stay, thanks! :)

**Pull request merged**